tools/test-backend

```diff
@@ -49,6 +57,19 @@ if __name__ == "__main__":
     test_runner = TestRunner()
     failures = test_runner.run_tests(suites, fatal_errors=options.fatal_errors)
 
+    templates_not_rendered = test_runner.get_shallow_tested_templates()
+    if templates_not_rendered:
+        missed_count = len(templates_not_rendered)
+        if options.no_shallow:
+            print("Templates not tested properly: {}".format(missed_count))
+            if options.verbose:
+                for template in templates_not_rendered:
+                    print('--- {}'.format(template))
+
+            failures = True
+        else:
+            print("Shallow tested templates: {}".format(missed_count))
```

Comment:
Maybe this output about shallow tested templates should be restricted to the `options.verbose` case?